database/migrations/sqls/20240627084826-pgwar-field-change-up.sql

```diff
@@ -0,0 +1,139 @@
+------ Table pgwar.field_change -----------------------------------------------------
+---------------------------------------------------------------------------------------------
+CREATE TABLE pgwar.field_change
+(
+    fk_project integer NOT NULL,
+    fk_source_info integer NOT NULL,
+    fk_source_tables_cell bigint,
```

Comment:
fk_source_tables_cell should also be NOT NULL

Reply:
I omitted NOT NULL because I couldn't assign a correct value as project_statements is currently missing a fk_subject_tables_cell column

Reply:
Ok I understand. But you are anyways passing 0 and not NULL when there is no value.

If you want to keep it nullable, we should set `UNIQUE NULLS NOT DISTINCT` instead of `UNIQUE`. 

See why: https://stackoverflow.com/questions/8289100/create-unique-constraint-with-null-columns

Reply:
Ok, I updated the code :)